support-frontend/assets/components/tooltip/Tooltip.tsx

```diff
@@ -164,28 +166,36 @@ const arrowTop = css`
 `;
 
 export type TooltipProps = {
-	promptText: string;
+	promptText?: string;
 	buttonLabel?: string;
 	children: React.ReactNode;
+	xAxisOffset?: number;
+	yAxisOffset?: number;
+	placement?: Placement;
```

Comment:
Given we don't use this anywhere else, we could avoid adding this extra complexity and just set the default to `'bottom'`.

Reply:
`Tooltip` is referenced in the `amountsAndBenefits` and in a story. The former could be enabled onto the `firstStepLanding` by the prop `showCancelTooltip`.  To preserve this compatibility I would like to keep these parameters untie we decide to create a PR to remove this feature from the `firstStepLanding` page (then they could be defaulted then).